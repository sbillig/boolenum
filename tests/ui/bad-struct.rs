use boolenum::BoolEnum;

#[derive(BoolEnum)]
struct StructInsteadOfEnum {
    yes: bool,
    no: bool,
}

fn main() {}
