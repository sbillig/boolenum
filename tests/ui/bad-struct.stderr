error: BoolEnum can only be used on enums.
 --> $DIR/bad-struct.rs:4:8
  |
4 | struct StructInsteadOfEnum {
  |        ^^^^^^^^^^^^^^^^^^^
